ReSharper.FSharp/test/data/features/service/typingAssist/XmlDoc 11 - Parens 01.fs.gold

```diff
@@ -0,0 +1,7 @@
+﻿// ${CHAR:<}
+module Module
+
+/// <summary>
+/// {caret}
+/// </summary>
+let f ((x, y)) = ()
```

Comment:
Is it correct that we don't create the param nodes here?

Reply:
Sure.

![image](https://user-images.githubusercontent.com/26364714/196241604-1e3e9353-3239-4e74-9bc5-4c3a1e8bf30d.png)